packages/app-store/routing-forms/__tests__/config.test.ts

```diff
@@ -83,9 +87,10 @@ describe("Query Builder Config", () => {
       assertCommonStructure(AttributesBaseConfig);
     });
 
-    it("should support multiselect_contains and multiselect_not_contains operators", () => {
-      expect(AttributesBaseConfig.operators).toHaveProperty("multiselect_contains");
-      expect(AttributesBaseConfig.operators).toHaveProperty("multiselect_not_contains");
+    it("should support multiselect_some_in operator for multiselect", () => {
```

Comment:
This operator is enabled only for AttributesBaseConfig and not for FormFieldsBaseConfig